config/templates/hashicorp_vault/lua/audit.lua

```diff
@@ -219,16 +219,23 @@ pathEnvToStandardEnv["prod"] = "production"
 pathEnvToStandardEnv["test"] = "test"
 pathEnvToStandardEnv["dev"] = "development"
 
+function parse_string(str, pattern)
+    local path_arr={}
```

Comment:
Make sure this PR follows the code style.

Reply:
Do not need use separate function to do parsing